### DISCUSSION TITLE
Using `skip_if_exists` option for folders whose names won't be known until after extension manipulation

### DISCUSSION THREAD

**Discussion by macintacos** [copier-org/copier#917] [Q&A]
I have a template which creates a folder after the user provides a value. The name of that folder is not directly assigned to the name that the user gives - the name is actually used to generate some other variables via extensions, and _those_ variables are then used throughout the project, one of which is used to name that folder.

I want to use the `skip_if_exists` option to ignore this folder on subsequent runs of the template. How would I do that? Couldn't find anything immediately obvious in the documentation.

Any help would be greatly appreciate - thanks!

**Comment by yajo** [comment]
Probably you'll get better support if you ask the plug-in creator.
